gems/smithy/lib/smithy/polishes/rubocop.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Polishes
+    # Runs RuboCop autocorrect-all on the generated projections.
+    class RuboCop < Polish
+      def polish(_artifact)
+        require 'rubocop'
+        puts "Running RuboCop --autocorrect-all on #{destination_root}"
+        rubocop = ::RuboCop::CLI.new
+        args = [
+          '--autocorrect-all',
+          '--display-only-fail-level-offenses',
+          destination_root
+        ]
+        rubocop.run(args)
```

Comment:
I really like this as a general approach - but I think we should not use it when running specs (it adds unessecary time + extra std out output).  I think theres probably a few ways we can do that.